### PR TITLE
added dnn tf apphub

### PR DESCRIPTION
**Pull request by purujitb**

### CODE REVIEW

File: fastestimator/dataset/data/breast_cancer.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+from fastestimator.dataset.numpy_dataset import NumpyDataset
+
+
+def load_data() -> Tuple[NumpyDataset, NumpyDataset]:
+    (X, y) = load_breast_cancer(True)
+    x_train, x_eval, y_train, y_eval = train_test_split(X, y, test_size=0.2)
+
+    x_train, x_eval = np.float32(x_train), np.float32(x_eval)
+
+    pdb.set_trace()
```
Comment (TortoiseHam) [inline fastestimator/dataset/data/breast_cancer.py]: pdb

File: fastestimator/dataset/data/breast_cancer.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    scaler = StandardScaler()
+    x_train = scaler.fit_transform(x_train)
```
Comment (TortoiseHam) [inline fastestimator/dataset/data/breast_cancer.py]: why are we doing this here instead of in pipeline op?

File: fastestimator/op/tensorop/loss/mean_squared_error.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+                 outputs: Union[None, str, Iterable[str]] = None,
+                 mode: Union[None, str, Iterable[str]] = None,
+                 average_loss: bool = True):
+        self.average_loss = average_loss
+        super().__init__(inputs=inputs, outputs=outputs, mode=mode)
+
+    def forward(self, data: List[Tensor], state: Dict[str, Any]) -> Tensor:
+        y_pred, y_true = data
+        loss = mean_squared_error(y_pred, y_true)
+        if self.average_loss:
```
Comment (TortoiseHam) [inline fastestimator/op/tensorop/loss/mean_squared_error.py]: I think loss averaging was moved into the backend function for the other loss ops

File: fastestimator/util/util.py [hunk 4]
```diff
@@ -178,6 +178,21 @@ def strip_suffix(target: Optional[str], suffix: Optional[str]) -> Optional[str]:
         return target[:-s_len]
     return target
 
+def is_number(number: str) -> bool:  # pylint: disable=invalid-name
```
Comment (TortoiseHam) [inline fastestimator/util/util.py]: where are you using this?

**Comment by vbvg2008** [comment]
closing this PR, I will make a new one with revised example
